docs/section5/Rule5-1.md

```diff
@@ -20,44 +20,42 @@
 
 - **Applicability Check 1:** ```if (rule-5-7  == PASS):```  
 
```

Comment:
Applicability check 1 (5-7) will have to be renumbered when all envelope TCDs are renumbered